LibertyDSNP/sdk-ts
# Features/update announcements to match spec 178933309

Problem
=======
We've updated the announcement spec since the SDK announcement module was written, so now the SDK is out of sync with the spec. Specifically, we're missing `createdAt` on all announcements.
[#178933309](https://www.pivotaltracker.com/story/show/178933309)

Solution
========
Added `createdAt` to all announcement types and updated tests accordingly

Change summary:
---------------
* Added `createdAt` to all DSNP announcement types
* Added an optional `createdAt` arg to DSNP factories
* Updated tests accordingly
* Also fixed GraphChange enum out of sync with spec

Steps to Verify:
----------------
1. Call a `create` method from the announcements module
1. Check that the resulting object has a `createdAt` field
